setup.py

```diff
@@ -5,21 +5,21 @@
     long_description = fh.read()
 
 setup(
-    name='python-ursa',
-    version='0.1.0',
+    name='python-indy-bls',
```

Comment:
Not entirely too sure, but could the name just be "indy-bls"? Might be a bit too verbose if the package has a prefix of python on the registry.

Reply:
Good catch -- I fixed it in one place not the other.  #rookie